scss/civicrm/contact/_tabs.scss

```diff
@@ -178,15 +180,11 @@
   /* override of jquery-ui.css that creates weird white background */
   .ui-widget-content {
     background: transparent;
+    border: 0 !important;
+    color: #222;
```

Comment:
what's this color? Shouldn't we have it in the palette?

Reply:
This again is stylelint change. I know that, this is not our palette color but, currently I have not modified anything to it to keep my scope limited to actual task.